### ISSUE TITLE
→ Accessibility: Components | Skeleton loading animation

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#1803]
### Description

In order to avoid failing https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide.html
we need to update the animation of the skeleton component with the following:

* Stop the animation if the skeleton is present more that 5s
* Allow to disable the animation via prefers-reduced-motion

Documentation:

* https://adrianroselli.com/2020/11/more-accessible-skeletons.html

### Tasks

- [ ] Update specs
- [ ] iOS ticket
- [ ] Android ticket
- [ ] Web ticket
